test/components/braveryPanelTest.js

```diff
@@ -268,6 +269,56 @@ describe('Bravery Panel', function () {
         .click(blockAdsOption)
         .waitForTextValue(adsBlockedStat, '2')
     })
+    it('blocks websocket tracking', function * () {
+      const url = Brave.server.url('websockets.html')
+      let time1 = 0
+      let time2 = 0
+      yield this.app.client
+        .waitForDataFile('adblock')
```

Comment:
we should really add a way to mock the adblock data so we don't have to wait for the download or worry about connection errors and changes in the adblock file contents